### ISSUE TITLE
Please make a better tutorial how to use it or new release .exe

### ISSUE THREAD

**Issue by Tulepeeker** [smokes/adobe-rpc#15]
I don't know anything about coding and don't understand the "How to use" and using a release doesn't satisfy me because I use Premiere.

**Comment by garrettsummerfi3ld** [comment]
What version of Python are you building on? I have a PR with how to build from source.

**Comment by garrettsummerfi3ld** [comment]
From my fork:

### Building binaries

Building the binaries from source requires `pyinstaller`

- Install `pip install -u pyinstaller`
- If you are building on Python 3.8+, run `pip install -u https://github.com/pyinstaller/pyinstaller/archive/develop.tar.gz`
- Build `pyinstaller --onefile .\rpc.py`

Let me know if there are any issues

**Comment by retconned** [comment]
> From my fork:
> 
> ### Building binaries
> Building the binaries from source requires `pyinstaller`
> 
> * Install `pip install -u pyinstaller`
> * If you are building on Python 3.8+, run `pip install -u https://github.com/pyinstaller/pyinstaller/archive/develop.tar.gz`
> * Build `pyinstaller --onefile .\rpc.py`
> 
> Let me know if there are any issues

Hey i just tried this and it didn't work its not displaying anything on discord

**Comment by garrettsummerfi3ld** [comment]
Is it showing anything?

**Comment by user-420** [comment]
Hey, I know this isn't my pull request but I get stuck in a "Run Adobe/Discord App" loop in console and nothing happens on the discord side of things. .
![image](https://user-images.githubusercontent.com/11807866/84114339-325b8c00-a9fa-11ea-885c-29140095dbab.png)

**Comment by garrettsummerfi3ld** [comment]
@user-420 is your adobe app supported?

**Comment by user-420** [comment]
@garrettsummerfi3ld Yeah, it should be, this happened both with photoshop and illustrator, I've also tried the EXE but it does nothing aswell.

Something that could (maybe) be causing issues is that I run both Discord and Discord PTB at the same time, because I have two accounts that I use for different things.

**Comment by garrettsummerfi3ld** [comment]
It shouldn't be it, but I'm not sure if that's the true answer.

**Comment by retconned** [comment]
> Is it showing anything?

i tried it a day after and it worked fine same thing also make sure everyone has this on :) 
![image](https://user-images.githubusercontent.com/38635147/84434238-f14cbd00-ac2f-11ea-803e-ed204d26e7d8.png)
